common/changes/office-ui-fabric-react/Create_OnMenuOpened_Callback_ComboBox_2017-10-20-22-22.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Adding OnMenuOpened callback",
+      "type": "minor"
```

Comment:
I also changed this to minor since you are adding to the comobox api

Reply:
thanks